scripts/docker/start-server.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+cd /standardebooks.org/web
+composer install
+
+ln -s /standardebooks.org/web/config/apache/standardebooks.test.conf /etc/apache2/sites-available/
+ln -s /standardebooks.org/web/config/php/fpm/standardebooks.org.ini /etc/php/*/cli/conf.d/
+ln -s /standardebooks.org/web/config/php/fpm/standardebooks.org.ini /etc/php/*/fpm/conf.d/
+ln -s /standardebooks.org/web/config/php/fpm/standardebooks.test.conf /etc/php/*/fpm/pool.d/
+a2ensite standardebooks.test
+/etc/init.d/apache2 start
+service php7.4-fpm restart
+tail -f /var/log/apache2/error.log
```

Comment:
Do we need to do this with the log file? Presumably this docker image works out of the box already

Reply:
We need to do something, otherwise the container finishes and exits 🙂 A lot of containers never suffer from this problem because they start an undaemonised server (`npm start` or equivalent), but here PHP and Apache both run in the background. A typical hack seems to be `tail -f /dev/null` in the foreground at the end of your script, so I’ve updated my script with that and a comment. https://github.com/standardebooks/web/pull/108/commits/2e28e5c65d6580e25d2aeffc5978eb5ef37f5bde